docs/examples/gestures/.gitignore

```diff
@@ -70,3 +70,8 @@
 !**/ios/**/default.pbxuser
 !**/ios/**/default.perspectivev3
 !/packages/flutter_tools/test/data/dart_dependencies_test/**/.packages
+
+lib/generated_plugin_registrant.dart
```

Comment:
I'm not sure if this file should be committed or not, I don't think there is any message on the file saying it shouldn't. but it does get generated on create right? what do other examples do?

Reply:
I believe this should be ignored, if you create a new project, using the newest Flutter with web enabled, you will notice this line on the generated gitignore file:

```
# Web related
lib/generated_plugin_registrant.dart
```